ui-admin/src/study/participants/participantList/ParticipantList.tsx

```diff
@@ -100,14 +100,14 @@ function ParticipantList({ studyEnvContext }: {studyEnvContext: StudyEnvContextT
   }, {
     id: 'familyName',
     header: 'Family name',
-    accessorKey: 'profile.familyName',
+    accessorFn: row => row.profile?.familyName,
     meta: {
       columnType: 'string'
     }
   }, {
     id: 'givenName',
     header: 'Given name',
-    accessorKey: 'profile.givenName',
+    accessorFn: row => row.profile?.givenName,
```

Comment:
driveby fixes for browser console warning `console.js:273 "lastLogin" in deeply nested key "participantUser.lastLogin" returned undefined. `